.github/workflows/ci.yml

```diff
@@ -434,12 +434,12 @@ jobs:
 
   upload_docs_release:
     name: "Upload release documentation"
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    if: github.event_name == 'push' #&& contains(github.ref, 'refs/tags')
     runs-on: ubuntu-latest
-    needs: [Release]
+    needs: [docs_build, build_test]
     steps:
       - name: Deploy the stable documentation
-        uses: pyansys/actions/doc-deploy-stable@v2
+        uses: pyansys/actions/doc-deploy-stable@v4
         with:
           cname: ${{ env.DOCUMENTATION_CNAME }}
           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@jorgepiloto 

I have patched the branch CI in here. This should update the version folder.

Reply:
Yep, that should do it =)